### PR TITLE
image: update image to 18400

### LINKED ISSUE #157: Update clear-containers-image to 18400
This new image contains fix to https://github.com/clearcontainers/agent/issues/130

### PR DESCRIPTION
**Pull request by erick0z**
This image contains relevant changes:
 - https://github.com/clearcontainers/agent/pull/131

Fixes #157

Signed-off-by: Erick Cardona <erick.cardona.ruiz@intel.com>

### CODE REVIEW

**Review by jcvenegas — APPROVED** [review]

**Review by gorozco1 — APPROVED** [review]
lgtm

File: clear-containers-image/debian.changelog [hunk 1]
```diff
@@ -1,3 +1,19 @@
+clear-containers-image (18400-36) stable; urgency=medium
+
+  * Update clear-containers-image from 18220 to 18400.
+
+version: 18280
```
Comment (jodh-intel) [inline clear-containers-image/debian.changelog]: All of this descriptive text needs to be indented as it is currently making this changelog file "invalid". See:

- https://www.debian.org/doc/debian-policy/#debian-changelog-debian-changelog

Note that you can check when the file is valid by doing:

```
$ sudo apt-get -y install dpkg-dev
$ dpkg-parsechangelog -l /path/to/your/changelog
```

If the `dpkg-parsechangelog` command outputs anything to `stderr`, the file is invalid (it seems that checking the exit code isn't reliable as the command attempts to recover and ignore bad input).

**Review by jodh-intel — CHANGES_REQUESTED** [review]

**Comment by erick0z** [comment]
@jodh-intel Fixed

File: clear-containers-image/debian.changelog [hunk 2]
```diff
@@ -1,3 +1,25 @@
+clear-containers-image (18400-36) stable; urgency=medium
+
+  * Update clear-containers-image from 18220 to 18400.
+
+  * version: 18280
+  Changes in package iptables (from 1.6.1-20 to 1.6.1-21):
```
Comment (jodh-intel) [inline clear-containers-image/debian.changelog:6]: Nit: I'd personally indent a little more (something like the following for maximum clarity - this is also still valid):

```
 * version: 18280

   Changes in package iptables (from 1.6.1-20 to 1.6.1-21):

       George T Kramer - iptables: Autospec creation for version 1.6.1

   https://download.clearlinux.org/releases/18280/clear/RELEASENOTES

```

**Review by jodh-intel — APPROVED** [review]

**Pull request merged**